index.html

```diff
@@ -107,6 +107,8 @@
         <li>Complete spec text
         <li>Designated reviewers have signed off on the current spec text
         <li>The ECMAScript editor has signed off on the current spec text
+        <li><a href="https://github.com/tc39/test262">Test262</a> acceptance tests have been written for mainline usage scenarios, in a PR
+        <li>If possible, there is an implementation in a polyfill or transpiler
```

Comment:
This is fine; stage 2 already conceptually means "this is when non-native implementations should ship"

Reply:
For context, I wanted to specifically give a role to transpilers/polyfills, if we're also specifically calling out native implementations for Stage 4; there's no intention to ignore them here. That's what's motivating the proposal for a change here.